.tflint.hcl

```diff
@@ -0,0 +1,5 @@
+plugin "aws" {
+    enabled = true
+    version = "0.13.2"
```

Comment:
This does not match the provider version in "required providers":
`  required_providers {
    datadog = {
      source  = "DataDog/datadog"
      version = ">= 2.10, < 3"
    }
    aws = {
      version = ">= 3.63"
    }`